tsconfig.json

```diff
@@ -6,6 +6,7 @@
         "sourceMap": false,
         "declaration": true,
         "module": "commonjs",
+        "experimentalDecorators": true,
```

Comment:
note that the tests have their own `tsconfig`, so this one shouldn't need to change